Sming/Libraries/OneWire/OneWire.cpp

```diff
@@ -124,7 +124,7 @@ OneWire::OneWire(uint8_t pin)
 
 void OneWire::begin()
 {
-	pinMode(pin, INPUT);
+	pinMode(pin, INPUT_PULLUP);
 	noPullup(pin);
```

Comment:
This looks like it will disable the pullup. To me this looks put here on purpose maybe the Arduino way is not to enable pullups?. Also be aware that if you enable pullups and also have pullups they will go in parallel making the equivalent smaller in value that together with line capacitance might not be appropiate for given sck frequency. 
I would not touch lib code if it is working an tested as it probably is consistent with Arduino way. After onewire::begin you can manually set the pins to pullup in your client code.